backend/bottari/src/main/java/com/bottari/teambottari/service/TeamBottariService.java

```diff
@@ -94,12 +96,74 @@ public Long create(
         }
     }
 
+    @Transactional
+    public void exit(
+            final Long id,
+            final String ssaid
+    ) {
+        final TeamBottari teamBottari = teamBottariRepository.findById(id)
+                .orElseThrow(() -> new BusinessException(ErrorCode.TEAM_BOTTARI_NOT_FOUND));
+        final List<TeamMember> allTeamMembers = teamMemberRepository.findAllByTeamBottariId(teamBottari.getId());
+        final TeamMember exitTeamMember = findExitTeamMember(ssaid, allTeamMembers);
```

Comment:
```suggestion
        final TeamBottari teamBottari = teamBottariRepository.findById(teamBottariId)
                .orElseThrow(() -> new BusinessException(ErrorCode.TEAM_BOTTARI_NOT_FOUND));
        final Member member = getMemberBySsaid(ssaid);
        final TeamMember exitTeamMember = findTeamMemberByTeamBottariAndMember(teamBottari, member);
```

팀 보따리를 찾고, 멤버를 찾은 후, 팀멤버를 찾아야. 각각의 예외를 더 명확하게 던질 수 있을 것 같은데 어때요?
- getMemberBySsaid(), findTeamMemberByTeamBottariAndMember() 이건 이미 있기도 하고!

Reply:
이렇게 하면 예외 메시지가 명확할거 같네요! 👍😊
그런데 지금도 충분히 로직으로 예외를 이해할 수 있을거같아요! 또한 이렇게하면 현재는 전체 팀 맴버를 불러오는 쿼리 1번으로 하는데, 지금 방식은 member, exitTeamMember, 그 외에 teamMember 찾는 쿼리 3번이 나가네요!